lambda-function-connect-rds-iam/example.js

```diff
@@ -0,0 +1,67 @@
+/* 
+Node.js code here.
+*/
+// ES6+ example
+import { Signer } from "@aws-sdk/rds-signer";
+import mysql from 'mysql2/promise';
+
+async function createAuthToken() {
+  // Define connection authentication parameters
+  const dbinfo = {
+
+    hostname: process.env.ProxyHostName,
+    port: process.env.Port,
+    username: process.env.DBUserName,
+    region: process.env.AWS_REGION,
+  
+  }
+
+  // Create RDS Signer object
+  const signer = new Signer(dbinfo);
+  
+  // Request authorization token from RDS, specifying the username
+  const token = await signer.getAuthToken(
+    {
+      username: process.env.DBUserName,
+    }
+  );
+  return token;
+}
+
+async function dbOps() {
+
+  try {
+      // Obtain auth token
+      const token = await createAuthToken();
+      // Define connection configuration
+      let connectionConfig = {
+        host: process.env.ProxyHostName,
+        user: process.env.DBUserName,
+        //password: token,
```

Comment:
What happened to passing the token as password? It should have been the most straightforward way to pass the token. If it didn't work, it's worth investigating why and leaving a comment about it if necessary.

Reply:
Removed in favor of token